### DISCUSSION TITLE
AttributeError

### DISCUSSION THREAD

**Discussion by sfc-gh-vbatra** [pyparsing/pyparsing#483] [General]
I am trying to test Lucene_Grammer example, but keep running into this error.
COLON, LBRACK, RBRACK, LBRACE, RBRACE, TILDE, CARAT = pp.Literal.using_each(":[]{}~^")
AttributeError: type object 'Literal' has no attribute 'using_each'

Please advise what am I doing wrong. I am using Pyparsing v 3.0.9

**Comment by ptmcg** [comment]
`using_each` will be part of the 3.1.0 release.

In the meantime, use the prior version of the lucene example, which should read something like:

    COLON, LBRACK, RBRACK, LBRACE, RBRACE, TILDE, CARAT = map(pp.Literal, ":[]{}~^")

Starting after 3.1.0, I'll start using a separate branch for release work instead of just pushing everything to master all the time, to avoid these kind of release race conditions.

**Comment by sfc-gh-vbatra** [comment]
That's what I used. Thank you very much for the clarification.
